src/targets/services/categorize.js

```diff
@@ -36,19 +33,26 @@ export const categorize = async () => {
     }
   } else {
     // Apply filters first
-    let filteredOperations = operations
-    if (filters.date) {
-      filteredOperations = filteredOperations.filter(
-        e =>
-          new Date(filters.date).valueOf() -
-            new Date(e.cozyMetadata.createdAt).valueOf() <=
-          0
-      )
+    let filtersToApply = {}
+
+    if (filters.minOperationDate) {
+      filtersToApply = Object.assign(filtersToApply, {
+        date: { $gt: filters.minOperationDate }
+      })
```

Comment:
Why not just 
```
const selector = filters.minOperationDate ? { date: { $gt: filters.minOperationDate }} : {}
...
.where(selector)
```

Reply:
This was meant to handle future filters but is not relevant yet